kodiswift/__init__.py

```diff
@@ -66,6 +66,9 @@ def func(*args, **kwargs):
     xbmcplugin = _Module(xbmcplugin)
     xbmcaddon = _Module(xbmcaddon)
     xbmcvfs = _Module(xbmcvfs)
+    for m in (xbmc, xbmcgui, xbmcplugin, xbmcaddon, xbmcvfs):
+        name = reversed(m.__name__.rsplit('.', 1)).next()
+        sys.modules[name] = m
```

Comment:
This is done to be able to fully mocks xbmc libraries even for non kodiswift's libraries